toolkit/tools/imagegen/attendedinstaller/primitives/enumfield/enumfield.go

```diff
@@ -112,9 +112,9 @@ func (n *EnumField) Draw(screen tcell.Screen) {
 	if (n.labelWidth > 0) && (n.labelWidth < labelWidth) {
 		labelWidth = n.labelWidth
 	}
-
-	_, drawnWidth := tview.Print(screen, n.label, x, y, labelWidth, tview.AlignLeft, n.labelColor)
-	x += drawnWidth
+	tview.Print(screen, n.label, x, y, labelWidth, tview.AlignLeft, n.labelColor)
+	// Always move to the edge to align input fields
+	x += n.labelWidth
```

Comment:
Since the width argument above is now always `labelWidth`, shouldn't we do `x += labelWidth` instead?

Reply:
No. Could you help me rephrase this comment? I wanted to explain it in the comment.

The idea is: label has a maximum width. We will only write as much as we can (usually labelWidth < n.labelWidth; n.labelWidth is a width of a label field). No matter how many characters we wrote, we always move the (drawing) pointer to the end of the label field. Otherwise input fields will not be aligned as different labels have different lengths.

Reply:
Got it. I guess we could just add a few details to your comment. Suggestion: "Regardless of printed width always move to the edge. Cannot move only by the printed width as this would leave the input fields unaligned".